packages/overlay/src/integrations/sentry/data/sentryDataCache.ts

```diff
@@ -34,7 +34,8 @@ class SentryDataCache {
 
   protected subscribers: Map<string, Subscription> = new Map();
 
-  protected contextLinesProvider: string = new URL(DEFAULT_SIDECAR_URL).origin + CONTEXT_LINES_ENDPOINT;
+  protected sidecarUrl: string = new URL(DEFAULT_SIDECAR_URL).origin;
+  protected contextLinesProvider: string = this.sidecarUrl + CONTEXT_LINES_ENDPOINT;
```

Comment:
Speaking of hypocrisy... 😅  -- Same comment above, we should probably use some sort of a URL builder at this point.

Reply:
yeah lets use `new URL()`